### DISCUSSION TITLE
How to apply a yaml file

### DISCUSSION THREAD

**Discussion by SpringHgui** [kubernetes-client/csharp#1135] [Q&A]
I have some yaml files, how to achieve the same effect with sdk like
```
kubectl apply -f xxxx.yaml
```
and
```
kubectl delete -f xxxx.yaml
```

**Comment by tg123** [comment]
you have to do it manually until kubectl like api finished

how apply works
https://github.com/kubernetes/kubectl/blob/master/pkg/cmd/apply/apply.go

**Reply by ffeliziani-tpmc** [reply]
forgive my ignorance (and the necrobump) but what does this mean? manually parse the yaml file and create the resources that way?

**Comment by SpringHgui** [comment]
🆗
